tools/dm/from-aurora.md

```diff
@@ -0,0 +1,188 @@
+
+复制并编辑 `{ansible deploy}/conf/task.yaml.example`，生成如下任务配置文件 `task.yaml`：
+
+```yaml
+# 任务名，多个同时运行的任务不能重名。
+name: "test"
+# 全量+增量 (all) 同步模式。
+task-mode: "all"
+# 下游 TiDB 配置信息。
+target-database:
+  host: "172.16.10.83"
+  port: 4000
+  user: "root"
+  password: ""
+
+# 当前数据同步任务需要的全部上游 MySQL 实例配置。
+mysql-instances:
+-
+  # 上游实例或者复制组 ID，参考 `inventory.ini` 的 `source_id` 或者 `dm-master.toml` 的 `source-id 配置`。
+  source-id: "mysql-replica-01"
+  # 需要同步的库名或表名的黑白名单的配置项名称，用于引用全局的黑白名单配置，全局配置见下面的 `black-white-list` 的配置。
+  black-white-list: "global"
+  # mydumper 的配置项名称，用于引用全局的 mydumper 配置。
+  mydumper-config-name: "global"
+
+-
+  source-id: "mysql-replica-02"
+  black-white-list: "global"
+  mydumper-config-name: "global"
+
+# 黑白名单全局配置，各实例通过配置项名引用。
+black-white-list:
+  global:
+    do-tables:                        # 需要同步的上游表的白名单。
+    - db-name: "test_db"              # 需要同步的表的库名。
+      tbl-name: "test_table"          # 需要同步的表的名称。
+
+# mydumper 全局配置，各实例通过配置项名引用。
+mydumpers:
+  global:
+    extra-args: "-B test_db -T test_table"  # 只导出 `test_db` 库中的 `test_table` 表，可设置 mydumper 的任何参数。
+```
+
+## 第 5 步：启动任务
+
+1. 进入 dmctl 目录 `/home/tidb/dm-ansible/resources/bin/`
+
+2. 执行以下命令启动 dmctl
+
+    ```bash
+    ./dmctl --master-addr 172.16.10.71:8261
+    ```
+
+3. 执行以下命令启动数据同步任务
+
+    ```bash
+    # `task.yaml` 是之前编辑的配置文件
+    start-task ./task.yaml
+    ```
+    
+    - 如果执行命令后的返回结果中不包含错误信息，则表明任务已经成功启动
+    - 如果包含以下错误信息，则表明上游 Aurora 用户可能拥有 TiDB 不支持的权限类型
+        ```json
+        {
+            "id": 4,
+            "name": "source db dump privilege chcker",
+            "desc": "check dump privileges of source DB",
+            "state": "fail",
+            "errorMsg": "line 1 column 285 near \"LOAD FROM S3, SELECT INTO S3 ON *.* TO 'root'@'%' WITH GRANT OPTION\" ...",
+            "instruction": "",
+            "extra": "address of db instance - pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com"
+        },
+        {
+            "id": 5,
+            "name": "source db replication privilege chcker",
+            "desc": "check replication privileges of source DB",
+            "state": "fail",
+            "errorMsg": "line 1 column 285 near \"LOAD FROM S3, SELECT INTO S3 ON *.* TO 'root'@'%' WITH GRANT OPTION\" ...",
+            "instruction": "",
+            "extra": "address of db instance - pingcap-1.h8emfqdptyc4.us-east-2.rds.amazonaws.com"
+        }
+        ```
+        此时可以选择以下两种处理方法中的任意一种进行处理后，再使用 `start-task` 尝试重新启动任务：
+        1. 为用于进行数据迁移的 Aurora 用户移除不被 TiDB 支持的不必要的权限
+        2. 如果能确保 Aurora 用户拥有 DM 所需要的权限，可以在 task.yaml 配置文件中添加如下顶级配置项以跳过启用任务时的前置权限检查
+            ```yaml
+            ignore-checking-items: ["dump_privilege", "replication_privilege"]
+            ```
+
+## 第 6 步：查询任务
+
+如需了解 DM 集群中是否存在正在运行的同步任务及任务状态等信息，可在 dmctl 内使用以下命令进行查询：
+
+```bash
+query-status
+```
+
+> 注意：
+> 如果查询命令的返回结果中包含以下错误信息，则表明在全量同步的 dump 阶段不能获得相应的 lock：
+>   ```bash
```

Comment:
I'm afraid that the code block cannot be displayed correctly on the PingCAP website, although it looks well on GitHub. We can keep it and have a try. If it doesn't work, could we change it into inline code?

Reply:
If it doesn't worker, we could change it into inline code.

Reply:
@csuzhangxc OK.